dev-tools/phpstan/baseline/offsetAccess.notFound.php

```diff
@@ -318,7 +318,7 @@
 ];
```

Comment:
...about https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7064#issuecomment-3233590502 
(i would love it to be a thread in first place)